README.md

```diff
@@ -50,7 +50,44 @@ This is the default Django workflow.
 
 To play around with it, you need some data.
 
-## CVEs
+## Fixtures
+
+A fixture file is availble for the `shared` app, located at `src/website/shared/fixtures/sample.json`.
+
+To load the data into the database:
+
+```console
+manage loaddata sample
+```
+
+Were `sample` is the name of fixture JSON file. Django will look inside the app folders for a fixture folders to match this name.
+
+### Recreating fixture files
+
+In case you want to recreate a fixture file:
+
+```console
+# Empty the database
+manage flush
+
+# Ingest CVE data
+manage ingest_bulk_cve --test
+
+# Register a Nix Channel in the database
+src/website/manage.py register_channel <null> nixos-unstable UNSTABLE
+
+# Ingest evaluation data
+manage ingest_manual_evaluation d616185828194210bfa0e51980d78a8bcd1246cc nixos-unstable evaluation.jsonl
```

Comment:
Is this the real SHA1? Just to be certain.

Reply:
It'd be nice if the SHA1 matches the true SHA1.

Reply:
For better context, I'm working on streamlining the setup of data for dev in [this PR](https://github.com/Nix-Security-WG/nix-security-tracker/pull/83) by introducing a `data_cache` directory. Next step will be to something similar for the evaluations ingestion, where the process will make always clear which hash generated the json file.

For the current README information, do you know which was the hash that produced the file at https://files.lahfa.xyz/private/evaluation.jsonl.zst.

Reply:
I believe it is `d616185828194210bfa0e51980d78a8bcd1246cc` according to my shell history logs.